docs/adv/troubleshooting/test_command.md

```diff
@@ -0,0 +1,161 @@
+
+#### PingLoad
+
+Same causes as PingMeasure test apply here.
+
+#### DirectConn
+
+- Your or your peer's port might not be publicly exposed.
+- Your or your peer's port might be behind a firewall.
+
+### Charon Relays
+
+#### PingRelay
+
+- Relay might be down or unaccessible because of other reasons.
+
+#### PingMeasureRelay
+
+- Relay might be under heavy load.
+- Your network latency might be high. Verify with the `charon test infra` tests.
+
+### Self
+
+#### Libp2pTCPPortOpenTest
+
+- There might be another process running on the designated port (tcp/3610 by default).
+- The process might have died.
+
+## Beacon
+
+#### Ping
+
+- Beacon node might not be started or is not reachable.
+
+#### PingMeasure
+
+- Beacon node might be too far away (geographically) from you.
+- Your network latency might be high. Verify with the `charon test infra` tests.
+
+#### Version
+
+- The beacon node version is not compatible with charon.
+
+#### IsSynced
+
+- Beacon node is not synced to the network.
+
+#### PeerCount
+
+- Beacon node does not have enough peers. This may result in slower fetching and broadcasting of slots and duties.
+
+#### PingLoad
+
+This is a load test, to enable it add the `--load-test` flag.
+
+Same causes as PingMeasure test apply here.
+
+#### Simulation
+
+This is a load test, to enable it add the `--load-test` flag.
+
+Same causes as PingMeasure test apply here and additionally:
+
+- The infrastructure on which the beacon node run (amount of RAM, disk IOPS) might be not enough to hold the number of validators supplied.
+
+## Validator
+
+#### Ping
+
+- Validator client might not be started or is not reachable.
+
+#### PingMeasure
+
+- Validator client might be too far away (geographically) from you. It is expected to run the validator client on the same machine.
+
+#### PingLoad
+
+Same causes as PingMeasure test apply here.
+
+## MEV
+
+#### Ping
+
+- MEV relay might not be started or is not reachable.
+
+#### PingMeasure
+
+- MEV relay might be too far away (geographically) from you.
+- Your network latency might be high. Verify with the `charon test infra` tests.
+
+#### CreateBlock
+
+Same causes as PingMeasure test apply here and additionally:
+
+- MEV relay might be too slow in block production.
+
+#### CreateMultipleBlocks
+
+Same causes as CreateBlock test apply here and additionally:
+
+## Infra
+
+#### DiskWriteSpeed
+
+- Your SSD/HDD write speed is too low. Minimum write speed is 500MBs, recommended write speed is 1000MBs.
+
+#### DiskWriteIOPS
+
+- Your SSD/HDD write operations per second are too low. Minimum write IOPS are 1000, recommended write IOPS are 2000.
+
+#### DiskReadSpeed
+
+- Your SSD/HDD read speed is too low. Minimum read speed is 500MBs, recommended read speed is 1000MBs.
+
+#### DiskReadIOPS
+
+- Your SSD/HDD read operations per second are too low. Minimum read IOPS are 1000, recommended read IOPS are 2000.
+
```

Comment:
We should add these minimums and recommendeds to the deployment best practices page, and potentially also make these link there so people can read more.

Reply:
Done